charts/datadog/templates/_container-agent.yaml

```
@@ -175,6 +175,9 @@
     {{- if eq (include "should-enable-container-image-collection" .) "true" }}
     - name: DD_CONTAINER_IMAGE_ENABLED
       value: "true"
+    {{- else }}
+    - name: DD_CONTAINER_IMAGE_ENABLED
+      value: "false"    
     {{- end }}
```

could we instead do:

```suggestion
    - name: DD_CONTAINER_IMAGE_ENABLED
      value: {{ include "should-enable-container-image-collection" . | quote }}
```